CMakeLists.txt

```diff
@@ -573,7 +573,7 @@ endif ()
 # and we need to call the executable from inside the CMake configuration
 #-----------------------------------------------------------------------------
 set (EXE_EXT "")
-if (WIN32 OR MINGW)
+if (WIN32)
```

Comment:
This needs checked - "if(MSVC)" might be preferable.

Reply:
yes, also EXE_EXT can be dropped